addons/stock/models/stock_move_line.py

```diff
@@ -824,6 +813,17 @@ def get_aggregated_properties(move_line=False, move=False):
 
         return aggregated_move_lines
 
+    def _get_aggregated_product_properties(self, move_line=False, move=False):
```

Comment:
Could you add `@api.model`, please?

To indicate `self` is never used.

Reply:
I already added the `@api.model`